Parsers/ASimProcessEvent/test/TrendMicro_VisionOne_ASimProcessCreate_DataTest.csv

```diff
@@ -0,0 +1,12 @@
+﻿Result
```

Comment:
Great work on this parser team. Please map threat fields too

Reply:
Thanks.
But we don't have threat fields available in logs.